packages/mobile/lib/pincodeManager.ts

```diff
@@ -0,0 +1,51 @@
+import 'capacitor-secure-storage-plugin';
+import { Plugins } from "@capacitor/core"
+
+const { SecureStoragePlugin } = Plugins
+
+function handle_keychain() {
+    
+    function set(key: string, pincode: string): Promise<boolean> {
+        return SecureStoragePlugin.set({ 
+            key, 
+            value: pincode 
+        }).catch(error => {
+            console.trace({ error })
+        })
+    }
+    
+    function get(key: string): Promise<string> {
+        return SecureStoragePlugin.get({ key })
+            .catch(error => {
+                console.trace({ error })
+        })
+    }
+
+    function remove(key: string): Promise<boolean> {
+        return SecureStoragePlugin.remove({ key })
+            .catch(error => {
+                console.trace({ error })
+        })
+    }
+
+    return {
+        /**
```

Comment:
Would be good to move the code comments to the function declaration.

Reply:
it's look weird to me too, that's because consumers can see this comments only when it's put on external closures functions